runtime/vm/bindnatv.cpp

```diff
@@ -301,7 +301,9 @@ static inlMapping mappings[] = {
 	{ "Java_sun_reflect_Reflection_getClassAccessFlags__Ljava_lang_Class_2", J9_BCLOOP_SEND_TARGET_INL_REFLECTION_GETCLASSACCESSFLAGS },
 #endif /* JAVA_SPEC_VERSION >= 11 */
 
-#if JAVA_SPEC_VERSION >= 22
+#if JAVA_SPEC_VERSION >= 24
```

Comment:
The comment on line 312 is now incorrect.

Reply:
Opened https://github.com/eclipse-openj9/openj9/pull/21257 to address it.